frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -210,43 +209,43 @@ export function PrintTestDeckScreen(): JSX.Element {
         return;
       }
 
+      const precinctId = precinctIds[printIndex.precinctIndex];
       await printer.print({ sides: 'two-sided-long-edge' });
       await logger.log(LogEventId.TestDeckPrinted, currentUserType, {
         disposition: 'success',
-        message: `Hand-marked paper ballot test deck printed as part of L&A package for precinct id: ${
-          precinctIds[printIndex.precinctIndex]
-        }`,
-        precinctId: precinctIds[printIndex.precinctIndex],
+        message: `Hand-marked paper ballot test deck printed as part of L&A package for precinct ID: ${precinctId}`,
+        precinctId,
       });
 
       if (printIndex.precinctIndex < precinctIds.length - 1) {
-        // wait 5s per ballot printed
-        // that's how long printing takes in duplex, no reason to get ahead of it.
-        await makeCancelable(sleep(numBallots * 5000));
+        await makeCancelable(sleep(numBallots * TWO_SIDED_PAGE_PRINT_TIME_MS));
         setPrintIndex({
           precinctIndex: printIndex.precinctIndex + 1,
-          component: 'tally',
+          component: 'TallyReport',
         });
       } else {
-        await makeCancelable(sleep(3000));
-        setPrintIndex(undefined);
+        await makeCancelable(sleep(numBallots * TWO_SIDED_PAGE_PRINT_TIME_MS));
```

Comment:
I believe we were previously forgetting to multiply by `numBallots` here

Reply:
Update: There was a reason for this! https://github.com/votingworks/vxsuite/pull/1863#discussion_r879666027

Switched back with tweaks for clarity: https://github.com/votingworks/vxsuite/pull/1863/commits/6802328453e09b05ab0cbf58c8cbb874a51df626